test/e2e/framework.go

```diff
@@ -1296,20 +1309,65 @@ func (data *TestData) gracefulExitAntreaAgent(covDir string, nodeName string) er
 		err = data.copyPodFiles(podName, "antrea-agent", antreaNamespace, antreaAgentCovFile, covDir)
 
 		if err != nil {
-			return fmt.Errorf("error when graceful exit Antrea agent: copy pod files out, error:%v", err)
+			return fmt.Errorf("error when graceful exit Antrea agent: copy antrea-agent coverage files out, error:%v", err)
 		}
+
 	}
 	return nil
 }
 
-// gracefulExitAntreaAgent copies the Antrea agent binary coverage data file out before terminating the Pod
-func (data *TestData) copyPodFiles(podName string, containerName string, nsName string, fileName string, covDir string) error {
-	fmt.Printf("Copying file %s from pod %s podName to '%s'", fileName, podName, covDir)
+//collectAntctlCovFiles collects coverage files for the antctl binary from the Pod and saves them to the coverage directory
+func (data *TestData) collectAntctlCovFiles(podName string, containerName string, nsName string, covDir string) error {
+	// copy antctl coverage files from Pod to the coverage directory
+	cmds := []string{"bash", "-c", "find . -maxdepth 1 -name 'antctl*.out' -exec basename {} ';'"}
+	stdout, stderr, err := data.runCommandFromPod(nsName, podName, containerName, cmds)
+	if err != nil {
+		return fmt.Errorf("error when finding the coverage files for antctl: <%v>, err: <%v>", stderr, err)
```

Comment:
please consider using a more consistent format in your error logs

you sometimes write "err", sometimes "error"; sometimes you print stderr contents (with or without "stderr:" prefix), sometimes you don't; sometimes you print the command that failed, sometimes you don't

Reply:
checked all the error message, and all changed to use stderr and err. Thanks for noticing this.